app/views/tasks/_task.html.haml

```diff
@@ -1,15 +1,23 @@
 %section.ccs-task{id: "task-#{task.id}"}
   %header.ccs-task__information
+    .ccs-task__statuses
+      - if task.complete?
+        %strong.govuk-tag.govuk-tag__success Completed
+      - if task.errors?
+        %strong.govuk-tag.govuk-tag__failure Errors
+      - if task.late?
```

Comment:
Completed tasks should never show as late - because of the logic in `late?` all completed tasks would eventually all display this label.

Reply:
another good catch, will update.

Reply:
completed tasks can no longer be late.

Reply:
:+1: